api/can_api_v2_definition.py

```diff
@@ -541,6 +552,69 @@ class CdcTransmissionLevelTimeseriesRow(base_model.APIBaseModel):
     )
 
 
+class CommunityLevels(base_model.APIBaseModel):
+    """Community levels for a region."""
+
+    cdcCommunityLevel: Optional[CommunityLevel] = pydantic.Field(
+        ...,
+        description=textwrap.dedent(
+            """
+    CDC Community level for county, as provided by the CDC.
+
+    Possible values:
+        - 0: Low
+        - 1: Medium
+        - 2: High
+
+    See https://www.cdc.gov/coronavirus/2019-ncov/science/community-levels.html
+    for details about how the Community Level is calculated and should be
+    interpretted.
+
+    Note that we provide two versions of the Community Level. One is called
+    canCommunityLevel which is calculated on a daily basis using CAN's data
+    sources and is available for states, counties, and metros.  The other is
+    called cdcCommunityLevel and is the raw Community Level published by the
+    CDC. It is only available for counties, and updates on a weekly basis.
+    """
+        ),
+    )
+
+    canCommunityLevel: Optional[CommunityLevel] = pydantic.Field(
+        ...,
+        description=textwrap.dedent(
+            """
+    Community level for region, calculated using the CDC definition but with CAN
+    data sources.
+
+    Possible values:
+        - 0: Low
+        - 1: Medium
+        - 2: High
+
+    See https://www.cdc.gov/coronavirus/2019-ncov/science/community-levels.html
+    for details about how the Community Level is calculated and should be
+    interpretted.
+
+    Note that we provide two versions of the Community Level. One is called
+    canCommunityLevel which is calculated on a daily basis using CAN's data
```

Comment:
Just to note for counties, this is calculated daily but I think it will only have a non `null` value once a week in the timeseries, (on the day that the hospital data is reported see screenshot). This makes sense/is expected I think, but it might be good to clarify that there won't be a `canCommunityLevel` value for every day in the API. 

<img width="336" alt="image" src="https://user-images.githubusercontent.com/55333380/161156817-c0c75415-b59e-48f0-9d1b-27d0d4f6febb.png">